docs/spec/resource-scopes.md

```diff
@@ -28,27 +32,28 @@ The following strings are permitted for the `targetScope` keyword: `'tenant'`, `
 It is important to set the target scope because it allows Bicep to perform validation that the resources declared in the `.bicep` file are permitted at that scope, and it also ensures that the correct type of scope is passed to the module when the module is referenced.
 
 ### Module 'scope' property
-When declaring a module, you can supply a property named `scope` to set the scope at which to deploy the module. If the module's target scope is the same as the parent's target scope, this property may be omitted.
+
+When declaring a module, you can supply an optional property named `scope` to set the scope at which to deploy the module. By default, bicep assumes the module will target the same scope as the parent if this property is omitted.
```

Comment:
Worth emphasizing that if parent doesn't have `targetScope`, resource group will be assumed?